src/js/edu-benefits/config/migrations.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash/fp';
+
+export function urlMigration(formId) {
+  // 0 -> 1, we have split the edu bundle into form-specific bundles with a new urlPrefix
+  // and this replaces the saved returnUrl if necessary
+  return (savedData) => {
+    const savedReturnUrl = _.get('return_url', savedData.metadata);
+
+    if (savedReturnUrl.includes(formId)) {
```

Comment:
Can we make this check case insensitive? I think we're likely storing lower case url prefixes, but it seems possible that we have a couple upper case ones (like 1990e versus 1990E).

`savedReturnUrl.toLowerCase().includes(formId.toLowerCase())` or something similar.

Reply:
Great point, `savedReturnUrl` has been made case insensitive.